setup.py

```diff
@@ -18,7 +18,7 @@
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
-    python_requires='>=3.7',
+    python_requires='>=3.6',
```

Comment:
This has gone down from 3.8 to 3.7 to 3.6. I'm just curious to know why - is there a reason that you need an older version, or are you just interested in supporting all currently supported pythons?

Reply:
My mac still has 3.6...

Reply:
given that we're not focusing on building in docker, that's a perfectly fine reason.